main.py

```diff
@@ -33,23 +35,23 @@ def main():
 
     credentials = JsonReader(CREDENTIALS_FILE_PATH)
 
-    meta_trader_factory = ContextFactory(production=True)
+    meta_trader_factory = ContextFactory(production=PRODUCTION)
     meta_trader = meta_trader_factory.create_context(json_settings.get_json_data(),credentials.get_json_data())
     meta_trader.connect()
 
     strategy = EmaStrategy(symbol,timeframe,EMA_SHORT,EMA_LONG)
     action_writer = strategy.get_action_writer()
     
-    symbol_factory = SymbolsFactory(production=True)
-    symbol = symbol_factory.create_symbol(symbol,timeframe, candles_mock_location=CANDLES_MOCK_LOCATION, ticks_mock_location=TICKS_MOCK_LOCATION) # Mock
+    symbol_factory = SymbolsFactory(production=PRODUCTION)
+    symbol = symbol_factory.create_symbol(symbol, timeframe, candles_mock_location=CANDLES_MOCK_LOCATION, ticks_mock_location=TICKS_MOCK_LOCATION) # Mock
```

Comment:
We should change this in the future to have only the simulator symbols class know about the location of the mocks.

Reply:
I think I abused passing parameters by assignment for this but open to modifying this if we follow the suggestion about the enum and multiple file sourcing above